docker-compose/goloop2goloop/Makefile

```diff
@@ -0,0 +1,12 @@
+build:
+	docker-compose build
+	
+run:
+	docker-compose up
+	
+clean:
+	docker-compose down -v --remove-orphans
```

Comment:
Could we use `docker-compose down -v --remove-orphans --rmi local` to remove the built images?

Reply:
@trmaphi If we remove the build image, then we lost the cached docker layer. The next time of building images is quite slow.